backend/src/main/java/pl/edu/agh/awiteks_backend/api/UserController.java

```diff
@@ -48,4 +52,15 @@ public void updateUser(@RequestBody User user) {
     public void removeUser(@PathVariable int id) {
         userService.remove(id);
     }
+
+    @Operation(summary = "Get all plants summary")
+    @GetMapping(value="/{id}/plants")
+    public List getAllPlantsSummary(@PathVariable int id){
+        Optional<User> user = this.userService.get(id);
+        return user.<List>map(value -> value
+                .getUserPlants()
+                .stream()
+                .map(PlantMapper::plantToPlantSummary)
+                .collect(Collectors.toList())).orElseGet(() -> Collections.singletonList(PlantSummary.builder().id(-1).name("luka").speciesName("luka").speciesName("luka").isFavourite(false).imgUrl("https://tatamariusz.pl/hans-christian-andersen-polny-kwiatek/#iLightbox[gallery3623]/0").build()));
```

Comment:
Ta cała logika powinna się znaleźć w serwisie. Pasuje w `UserService` stworzyć metodę `getAllPlantsSummary` i tam powinno się to wykonywać, a w controlerze robicie tylko coś takiego
```java
@Operation(summary = "Get all plants summary")
@GetMapping(value="/{id}/plants")
public List getAllPlantsSummary(@PathVariable int id){
   userService.getAllPlantsSummary(id)
}
```